`VERSION_RE` fails to match version info from `mysql` installed with `brew` on OSX High Sierra

### What action do you want to perform
Run tests with `mysql_proc` fixture.

OS: OS X High Sierra
Python Version: 3.6.1
MySQL Version: 5.7.23 (Homebrew)
pytest-mysql Version: 2.0.0

version output:
```
$mysqld --version
mysqld  Ver 5.7.23 for osx10.13 on x86_64 (Homebrew)
```

### What are the results
`VERSION_RE.match(version_output)` returns `None` where version number `str` is expected.
```
self = <pytest_mysql.executor.MySQLExecutor: "
        /..." 0x104790fd0>

    def version(self):
        """Read MySQL's version."""
        version_output = subprocess.check_output(
            [self.mysqld, '--version']
        ).decode('utf-8')
>       return self.VERSION_RE.match(version_output).groupdict()['version']
E       AttributeError: 'NoneType' object has no attribute 'groupdict'

../../../.pyenv/versions/mmt-server-promotion2/lib/python3.6/site-packages/pytest_mysql/executor.py:80: AttributeError
```

### What are the expected results
Version number is parsed and tests run without errors.

--- a/src/pytest_mysql/executor.py
+++ b/src/pytest_mysql/executor.py
@@ -77,7 +77,7 @@ class MySQLExecutor(TCPExecutor):
         version_output = subprocess.check_output(
             [self.mysqld, '--version']
         ).decode('utf-8')
-        return self.VERSION_RE.match(version_output).groupdict()['version']
+        return self.VERSION_RE.search(version_output).groupdict()['version']
 
     def implementation(self):
         """Detect MySQL Implementation."""

--- a/tests/test_executor.py
+++ b/tests/test_executor.py
@@ -23,8 +23,8 @@ from pytest_mysql.executor import MySQLExecutor, MySQLUnsupported
         ),
         '10.1.30'
     ),
-    ('mysqld  Ver 8.0.12 for macos10.13 on x86_64 (MySQL Community Server - GPL)', '8.0.12'),
-    ('mysqld  Ver 5.7.23 for osx10.13 on x86_64 (Homebrew)', '5.7.23')
+    (b'mysqld  Ver 8.0.12 for macos10.13 on x86_64 (MySQL Community Server - GPL)', '8.0.12'),
+    (b'mysqld  Ver 5.7.23 for osx10.13 on x86_64 (Homebrew)', '5.7.23')
 ))
 def test_version_check(verstr, version, tmpdir_factory):
     """Test executor's version property."""

--- a/tests/test_executor.py
+++ b/tests/test_executor.py
@@ -22,7 +22,23 @@ from pytest_mysql.executor import MySQLExecutor, MySQLUnsupported
             b'for debian-linux-gnu on x86_64 (Ubuntu 17.10)'
         ),
         '10.1.30'
-    )
+    ),
+    (b'mysql 8.0.12', '8.0.12'),
+    (
+        (
+            b'Ver 8.0.12'
+            b' for macos10.13 on x86_64 (MySQL Community Server - GPL)'
+        ),
+        '8.0.12'
+    ),
+    (b'mysql 5.7.23', '5.7.23'),
+    (
+        (
+            b'mysqld  Ver 5.7.23 '
+            b'for osx10.13 on x86_64 (Homebrew)'
+        ),
+        '5.7.23'
+    ),
 ))
 def test_version_check(verstr, version, tmpdir_factory):
     """Test executor's version property."""
@@ -61,7 +77,23 @@ def test_version_check(verstr, version, tmpdir_factory):
             b'for debian-linux-gnu on x86_64 (Ubuntu 17.10)'
         ),
         'mariadb'
-    )
+    ),
+    (b'mysql 8.0.12', 'mysql'),
+    (
+        (
+            b'Ver 8.0.12'
+            b' for macos10.13 on x86_64 (MySQL Community Server - GPL)'
+        ),
+        'mysql'
+    ),
+    (b'mysql 5.7.23', 'mysql'),
+    (
+        (
+            b'mysqld  Ver 5.7.23 '
+            b'for osx10.13 on x86_64 (Homebrew)'
+        ),
+        'mysql'
+    ),
 ))
 def test_implementation(verstr, implementation, tmpdir_factory):
     """Check detecting implementation."""

--- a/tests/test_executor.py
+++ b/tests/test_executor.py
@@ -23,22 +23,8 @@ from pytest_mysql.executor import MySQLExecutor, MySQLUnsupported
         ),
         '10.1.30'
     ),
-    (b'mysql 8.0.12', '8.0.12'),
-    (
-        (
-            b'mysqld  Ver 8.0.12'
-            b' for macos10.13 on x86_64 (MySQL Community Server - GPL)'
-        ),
-        '8.0.12'
-    ),
-    (b'mysql 5.7.23', '5.7.23'),
-    (
-        (
-            b'mysqld  Ver 5.7.23 '
-            b'for osx10.13 on x86_64 (Homebrew)'
-        ),
-        '5.7.23'
-    ),
+    ('mysqld  Ver 8.0.12 for macos10.13 on x86_64 (MySQL Community Server - GPL)', '8.0.12'),
+    ('mysqld  Ver 5.7.23 for osx10.13 on x86_64 (Homebrew)', '5.7.23')
 ))
 def test_version_check(verstr, version, tmpdir_factory):
     """Test executor's version property."""

--- a/tests/test_executor.py
+++ b/tests/test_executor.py
@@ -26,7 +26,7 @@ from pytest_mysql.executor import MySQLExecutor, MySQLUnsupported
     (b'mysql 8.0.12', '8.0.12'),
     (
         (
-            b'Ver 8.0.12'
+            b'mysqld  Ver 8.0.12'
             b' for macos10.13 on x86_64 (MySQL Community Server - GPL)'
         ),
         '8.0.12'

--- a/tests/test_executor.py
+++ b/tests/test_executor.py
@@ -23,8 +23,20 @@ from pytest_mysql.executor import MySQLExecutor, MySQLUnsupported
         ),
         '10.1.30'
     ),
-    (b'mysqld  Ver 8.0.12 for macos10.13 on x86_64 (MySQL Community Server - GPL)', '8.0.12'),
-    (b'mysqld  Ver 5.7.23 for osx10.13 on x86_64 (Homebrew)', '5.7.23')
+    (
+        (
+            b'mysqld  Ver 8.0.12 for macos10.13 on x86_64 '
+            b'(MySQL Community Server - GPL)'
+        ),
+        '8.0.12'
+    ),
+    (
+        (
+            b'mysqld  Ver 5.7.23 for osx10.13 on x86_64 '
+            b'(Homebrew)'
+        ),
+        '5.7.23'
+    )
 ))
 def test_version_check(verstr, version, tmpdir_factory):
     """Test executor's version property."""

